pvlib/test/test_modelchain.py

```diff
@@ -263,21 +263,16 @@ def constant_spectral_loss(mc):
     mc.spectral_modifier = 0.9
 
 @requires_scipy
-@pytest.mark.parametrize('spectral_model, expected', [
-    ('sapm', [182.338436597, -2.00000000e-02]),
-    pytest.mark.xfail(raises=NotImplementedError)
-    (('first_solar', [179.371460714, -2.00000000e-02])),
-    ('no_loss', [181.604438144, -2.00000000e-02]),
-    (constant_spectral_loss, [163.061569511, -2e-2])
-])
-def test_spectral_models(system, location, spectral_model, expected):
+@pytest.mark.parametrize('spectral_model', ['sapm', 'first_solar', 'no_loss'])
```

Comment:
please keep ``constant_spectral_loss`` in the list of parameters. This tests that a user-supplied function can be passed in as an argument. To be clear, the list should be: ``['sapm', 'first_solar', 'no_loss', constant_spectral_loss]``

Reply:
constant_spectral_loss --> constant_loss ?

Reply:
I put the keyword back.  I tend to agree with @adriesse about 'loss' but the terminology is common.